actix-template-cli/src/commands/contacts.rs

```diff
@@ -0,0 +1,272 @@
+use super::configure::ActixTemplateConfiguration;
+use crate::{errors::DefaultError, ui::get_cancelable_render_config};
+use actix_web_starter_client::{
+    apis::contacts_api,
+    models::{self, Contact},
+};
+use clap::{Args, Subcommand};
+use inquire::validator::Validation;
+use tabled::settings::Style;
+
+#[derive(Subcommand)]
+pub enum ContactCommands {
+    Create,
+    Delete(DeleteContact),
+    Edit(EditContact),
+    View(ViewContact),
+    List,
+}
+
+#[derive(Args)]
+pub struct DeleteContact {
+    /// The id of the contact to delete
+    pub id: Option<String>,
+}
+
+#[derive(Args)]
+pub struct EditContact {
+    /// The id of the contact to edit
+    pub id: Option<String>,
+}
+
+#[derive(Args)]
+pub struct ViewContact {
+    /// The id of the contact to view
+    pub id: Option<String>,
+}
+
+pub async fn create_contact_cmd(config: ActixTemplateConfiguration) -> Result<(), DefaultError> {
+    let first_name = inquire::Text::new("Enter the first name of the contact:")
+        .with_render_config(get_cancelable_render_config("No First Name Entered"))
+        .prompt()?;
+    let last_name = inquire::Text::new("Enter the last name of the contact:")
+        .with_render_config(get_cancelable_render_config("No Last Name Entered"))
+        .prompt()?;
+    let result = contacts_api::create_contact(
+        &config.clone().into(),
+        contacts_api::CreateContactParams {
+            organization: config.org_id,
+            create_contact_req_payload: actix_web_starter_client::models::CreateContactReqPayload {
+                first_name,
+                last_name,
+            },
+        },
+    )
+    .await?
+    .entity
+    .ok_or_else(|| DefaultError::new("No entity returned from API for create contact"))?;
+
+    match result {
+        contacts_api::CreateContactSuccess::Status201(_) => {
+            println!("Contact created successfully");
```

Comment:
Would be nice to print the id here so that users can copy paste it into future commands instead of having to go through the list step.